model/Manager.php

```diff
@@ -2,9 +2,32 @@
 
 namespace EmilieSchott\BlogPHP\Model;
 
-class Manager {
-    protected function dbConnect() {
-        $db=new \PDO('mysql:host=localhost;dbname=p5_blog_php;port=3308;charset=utf8', 'root', '', array(\PDO::ATTR_ERRMODE=>\PDO::ERRMODE_EXCEPTION));
-        return $db;
+abstract class Manager {
+    protected $db;
+    
+    public function __construct() {
```

Comment:
What happens if you create two managers that extend this class in the same run? Like a PostManager and a Comment Manager, for instance? ;)

Reply:
Hm... Several $db, that's wrong ... I didn't think about it, thanks !